lib/data/tweet_votes_record.py

```diff
@@ -0,0 +1,48 @@
+"""
+tweets_vote_record.py
+------------------------
+投票を保存するクラスが入っている。
+"""
+import threading
+from typing import Dict, Optional
+
+from lib.data.tweet_vote import TweetVote
+
+
+class TweetsVoteRecord:
+    """
+    進行中の投票をIDと関連付けて持っておくクラス。
+    処理は全てスレッドセーフ。
+    """
+    def __init__(self):
+        self.pending_tweets_list: Dict[int, TweetVote] = {}
+        self.thread_lock: threading.Lock = threading.Lock()
+
+    def add(self, tweet_id: int, content: TweetVote):
+        """
+        投票を登録する。
+        :param tweet_id: 投票のID。
+        :param content: 投票。
+        """
+        with self.thread_lock:
+            self.pending_tweets_list[tweet_id] = content
+
```

Comment:
maybe dup check is helpful?

```suggestion
        with self.thread_lock:
            if tweet_id in self.pending_tweets_list: 
                # return error which represents duplicated tweet_id was passed
            self.pending_tweets_list[tweet_id] = content

```

Reply:
3b35cf5